content/en/blog/_posts/2022-10-05-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,117 @@
+| 37     | Kubelet supports insecure TLS ciphersuites                                  | [#81145](https://github.com/kubernetes/kubernetes/issues/81145) | closed but fix needed for [#91444](https://github.com/kubernetes/kubernetes/issues/91444) (see [this comment](https://github.com/kubernetes/kubernetes/issues/81145#issuecomment-630291221))                                           |
+
+## Call to Action
+
+Many of the 37 findings identified were fixed by work from
+our community members over the last 3 years. However, we still have some work
+left to do. Here's a breakdown of remaining work with rough estimates on
+time commitment, complexity and benefits to the ecosystem on fixing
+these pending issues.
+
+{{<note>}}
+
+Anything requiring a KEP (Kubernetes Enhancement Proposal) is considered
+_high_ time commitment and _high_ complexity. Benefits to Ecosystem are
+roughly equivalent to risk of keeping the finding unfixed which is
+determined by Severity Level + Likelihood of a successful vulnerability
+exploit. These estimates and values in the table below are the authors' 
+personal opinion. An individual or end users' threat model may rate the
+benefits to fix a particular issue higher or lower.
+
+{{</note>}}
+
+| Title                                                                            | Issue                                                           | Time Commitment | Complexity | Benefit to Ecosystem |
+|----------------------------------------------------------------------------------|-----------------------------------------------------------------|-----------------|------------|----------------------|
+| Kubernetes does not facilitate certificate revocation                            | [#81111](https://github.com/kubernetes/kubernetes/issues/81111) | High            | High       | Medium               |
+| Use of InsecureSkipVerify and other TLS weaknesses                               | [#81119](https://github.com/kubernetes/kubernetes/issues/81119) | High            | High       | Medium               |
+| Kubectl can cause a local Out Of Memory error with a malicious Pod specification | [#81123](https://github.com/kubernetes/kubernetes/issues/81123) | Medium          | Medium     | Medium               |
+| Improper fetching of PIDs allows incorrect cgroup movement                       | [#81124](https://github.com/kubernetes/kubernetes/issues/81124) | Medium          | Medium     | Medium               |
+| kubelet liveness probes can be used to enumerate host network                    | [#81129](https://github.com/kubernetes/kubernetes/issues/81129) | High            | High       | Medium               |
+| API Server supports insecure TLS ciphersuites                                    | [#81145](https://github.com/kubernetes/kubernetes/issues/81145) | Medium          | Medium     | Low                  |
```

Comment:
I'd strongly argue the benefit to the ecosystem is critically, indispensably high.  Without taking the extra effort to fix this in production systems your apiserver is effectively and practically insecure. I recall [this xkcd](https://xkcd.com/2347) and replace its point with all modern security balancing on  _very flawed and fragile assumptions about the out of the box security of TLS_.

Reply:
Based on my understanding of thee issue, I'd have it as a Low, given that cluster operators have to opt in to running a more vulnerable cipher / cipher suite. People who don't know much about TLS are unlikely to turn these on.

We could add some details about [not doing[ that to https://kubernetes.io/docs/concepts/security/security-checklist/